assets/components/DropdownFilterButton.jsx

```diff
@@ -4,7 +4,14 @@ import {gettext} from 'utils';
 import classNames from 'classnames';
 
 const getActiveFilterLabel = (filter, activeFilter, isActive) => {
-    return isActive ? activeFilter[filter.field][0] : gettext(filter.label);
+    let filterLabel = '';
+    if (isActive) {
+        filterLabel = filter.transform  ? gettext(filter.transform(activeFilter[filter.field][0])) : gettext(activeFilter[filter.field][0]);
```

Comment:
imo just returning from here and the else block instead of using a variable for that would make it a bit simpler